elements/pfe-sass/mixins/_custom-properties.scss

```diff
@@ -0,0 +1,172 @@
+// New broadcast mixin, for use with content components
+//  1st arg:  lookup surface value from theme
+//  2nd arg:  list of broadcast variable - optional
+//  3rd arg:  identify the color set, default is surface - optional
+//
+//  usage:
+//    :host {
+//      #{pfe-set-broadcast(dark)};
+//    }
+//  result:
+//    :host {
+//      --pfe-broadcasted--color--text:            #{pfe-color(text--on-dark)};
+//      --pfe-broadcasted--color--link:            #{pfe-color(link--on-dark)};
+//      --pfe-broadcasted--color--link--visited:   #{pfe-color(link--visited--on-dark)};
+//      --pfe-broadcasted--color--link--hover:     #{pfe-color(link--hover--on-dark)};
+//      --pfe-broadcasted--color--link--focus:     #{pfe-color(link--focus--on-dark)};
+//    }
+
+@mixin pfe-set-broadcast($theme, $broadcast: $BROADCAST-VARS) {
```

Comment:
what do you think about the name: `@mixin pfe-set-broadcast-theme`  since its now a grouping of all the broadcast variables, and the required argument is a theme?

Reply:
Applied in b08ff9947d78d551a6d86b5db96c0a4f84c43057